Dockerfile

```diff
@@ -97,11 +97,11 @@ USER root
 
 ### JAVASCRIPT
 
-# Install Node 14.0 and npm v7
+# Install Node 14.0 and npm v8
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - \
   && apt-get install -y --no-install-recommends nodejs \
   && rm -rf /var/lib/apt/lists/* \
-  && npm install -g npm@v7.21.0 \
+  && npm install -g npm@v8.3.0 \
```

Comment:
```suggestion
```

Reply:
The docker win is nice but I think we still want to keep this explicit so we can update to new patch releases. Otherwise we're at the mercy of docker caching the node + npm version.